src/GeneralStateTestsFiller/stCreateTest/CreateCollisionResultsFiller.yml

```diff
@@ -0,0 +1,188 @@
+CreateCollisionResults:
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: 0x100000000
+    currentNumber: "1"
+    currentTimestamp: "1000"
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  _info:
+    comment: Ori Pomerantz qbzzt1@gmail.com
+
+
+  pre:
+ 
+    # Same address as CREATE creates. 
+    8af6a7af30d840ba137e8f3f34d54cfb8beba6e2:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+          [[0]] 0x001D
+        }
+      nonce: '0'
+      storage:
+        0x0: 0x60A7
+
+
+    # Same address as CREATE2 creates
+    40f1299359ea754ac29eb2662a1900752bf8275f:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+          [[0]] 0x001D
+        }
+      nonce: '0'
+      storage:
+        0x0: 0x60A7
+
+
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code:   | 
+        {
+
+          ; Variables are 0x20 bytes (= 256 bits) apart, except for
+          ; code buffers that get 0x100 (256 bytes)
+          (def 'creation          0x100)
+          (def 'contractCode      0x200)
+          (def 'constructorCode   0x300)
+          (def 'contractLength    0x520)
+          (def 'constructorLength 0x540)
+          (def 'addr1             0x600)
+          (def 'callRet           0x640)
+
+          ; Other constants
+          (def 'NOP 0)   ; No OPeration
+
+          ; Understand the input.
+          [creation]       (shr $ 0 248)
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Create the contract and a constructor to pass to CREATE[2]
+          ;
+
+          [constructorLength] 
+            (lll {   ; successful constructor
+               (return contractCode
+                  ; write the contract to constructor memory 
+                  (lll
+                     (sstore 0 0xFF)
+                     contractCode
+                  ) ; inner lll
+               )  } ; return contractCode (lll ...)
+                constructorCode
+            )       ; outer lll, if successful
+
+          [contractLength]
+            (lll
+               (sstore 0 0xFF)
+               contractCode
+            )
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Actually create the contract
+          ;
+          (if (= @creation 1)
+            [addr1] (create 0 constructorCode @constructorLength)
+            [addr1] (create2 0 constructorCode @constructorLength 0x5a17)
+          )
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; RETURNDATASIZE should be zero
+          ;
+
+          [[0x10]] (returndatasize)
```

Comment:
+ returndatacopy, extcodesize, extcodecopy and pc

Reply:
1. When RETURNDATASIZE is zero do you still need me to have RETURNDATACOPY?
2. I already checked EXTCODE in the regular create test. Do you want me to also check it here?
3. What do you want me to check with PC?

Reply:
Yes. Check everything regardles of the result of other checks

Extcode opcodes also to be check in all cases with create (success/oog in constructor/revert in constructor/collision)

Reply:
Done